docs/_policies/leavingGSB.md

```diff
@@ -62,14 +72,18 @@ For other systems across campus, the data retention and offboarding policies may
 
 ## AWS
 
-If you use [Athena](../services/athena.html), [Cloud Forest](../services/cloudForest.html) or a custom built cloud solution, it is your responsibility to transition data off of these systems before leaving the school.
+If you've set up a AWS through Universtiy Information Technology, please go to [their cloud storage page](https://stanford.service-now.com/it_services?id=sc_category&sys_id=e6cc5a8813b9aa0019813598d144b09b){target="_blank"} to make an inquiry.
 
-{% include warning.html content="Cloud Forest is not designed for storing data, and any important results should be immediately transferred off of these ephemeral machines." %}
+If you use Athena, Cloud Forest or a custom built cloud solution, it is your responsibility to transition data off of these systems before leaving the school.
+
+!!! warning
+    Cloud Forest is not designed for storing data, and any important results should be immediately transferred off of these ephemeral machines.
 
 The following may be decommissioned and purged without notice after you leave the GSB:
+
 * Cloud Forest instances and data
 * Custom built applications, services or websites
-* Accounts provisioned specifically for you
+* Accounts provisioned specifically for you by DARC
 * Unclaimed data on S3 or EBS volumes
 
 ### Sponsorship
```

Comment:
This doesn't seem relevant to this page? I think we should cut this whole section